core/ir/aarch64/codec.txt

```diff
@@ -294,1336 +294,1347 @@ x----------xxxxx----------------  wx16       # W/X register (or WZR/XZR)
-0x101110xx1xxxxx010101xxxxxxxxxx  n   543      urshl            dq0 : dq5 dq16 bhsd_sz
-01111110111xxxxx010101xxxxxxxxxx  n   543      urshl             d0 : d5 d16
-0x1011110xxxxxxx001001xxxxxxxxxx  n   544      urshr            dq0 : dq5 bhsd_immh_sz immhb_shf
-0111111101xxxxxx001001xxxxxxxxxx  n   544      urshr             d0 : d5 immhb_shf
-0x10111010100001110010xxxxxxxxxx  n   545    ursqrte            dq0 : dq5 sd_sz
-0111111101xxxxxx001101xxxxxxxxxx  n   546      ursra             d0 : d5 immhb_shf
-0x1011110xxxxxxx001101xxxxxxxxxx  n   546      ursra            dq0 : dq5 bhsd_immh_sz immhb_shf
-0x101110xx1xxxxx010001xxxxxxxxxx  n   547       ushl            dq0 : dq5 dq16 bhsd_sz
-01111110111xxxxx010001xxxxxxxxxx  n   547       ushl             d0 : d5 d16
-0010111100xxxxxx101001xxxxxxxxxx  n   548      ushll             q0 : d5 bhsd_immh_sz immhb_0shf
-0110111100xxxxxx101001xxxxxxxxxx  n   549     ushll2             q0 : q5 bhsd_immh_sz immhb_0shf
-0111111101xxxxxx000001xxxxxxxxxx  n   550       ushr             d0 : d5 immhb_shf
-0x1011110xxxxxxx000001xxxxxxxxxx  n   550       ushr            dq0 : dq5 bhsd_immh_sz immhb_shf
-0111111000100000001110xxxxxxxxxx  n   551     usqadd             b0 : b5
-0111111001100000001110xxxxxxxxxx  n   551     usqadd             h0 : h5
-0111111010100000001110xxxxxxxxxx  n   551     usqadd             s0 : s5
-0111111011100000001110xxxxxxxxxx  n   551     usqadd             d0 : d5
-0x101110xx100000001110xxxxxxxxxx  n   551     usqadd            dq0 : dq5 bhsd_sz
-0111111101xxxxxx000101xxxxxxxxxx  n   552       usra             d0 : d5 immhb_shf
-0x1011110xxxxxxx000101xxxxxxxxxx  n   552       usra            dq0 : dq5 bhsd_immh_sz immhb_shf
-00101110xx1xxxxx001000xxxxxxxxxx  n   553      usubl             q0 : d5 d16 bhs_sz
-01101110xx1xxxxx001000xxxxxxxxxx  n   554     usubl2             q0 : q5 q16 bhs_sz
-00101110xx1xxxxx001100xxxxxxxxxx  n   555      usubw             q0 : q5 d16 bhs_sz
-01101110xx1xxxxx001100xxxxxxxxxx  n   556     usubw2             q0 : q5 q16 bhs_sz
-0x001110xx0xxxxx000110xxxxxxxxxx  n   557       uzp1            dq0 : dq5 dq16 bhsd_sz
-0x001110xx0xxxxx010110xxxxxxxxxx  n   558       uzp2            dq0 : dq5 dq16 bhsd_sz
-11010101000000110010000001011111  n   559        wfe                :
-11010101000000110010000001111111  n   560        wfi                :
-11010101000000110010000011111111  n   561    xpaclri                :
-00001110xx100001001010xxxxxxxxxx  n   562        xtn             d0 : d5 bhsd_sz
-01001110xx100001001010xxxxxxxxxx  n   563       xtn2             q0 : q5 bhsd_sz
-11010101000000110010000000111111  n   564      yield                :
-0x001110xx0xxxxx001110xxxxxxxxxx  n   565       zip1            dq0 : dq5 dq16 bhsd_sz
-0x001110xx0xxxxx011110xxxxxxxxxx  n   566       zip2            dq0 : dq5 dq16 bhsd_sz
+0101111011100000101110xxxxxxxxxx  n   6           abs             d0 : d5
```

Comment:
Can't tell what changed here: an extra space?  Normally the diff would identify that as the only change on the line: are there other changes?

Reply:
See later comment https://github.com/DynamoRIO/dynamorio/pull/5235#discussion_r761123121